pkg/executor/aggregate/agg_hash_final_worker.go

```diff
@@ -73,72 +94,82 @@ func (w *HashAggFinalWorker) initBInMap() {
 	}
 }
 
-func (w *HashAggFinalWorker) consumeIntermData(sctx sessionctx.Context) (err error) {
+func (w *HashAggFinalWorker) handleFirstInput(input *aggfuncs.AggPartialResultMapper) {
```

Comment:
How about if the size of `w.partialResultMap` is 0, then just treat it as the first input, so we don't need maintain the status of `isFirstInput`, and `handleFisrtInput` can be removed which can make the code easier to read.

Reply:
> How about if the size of `w.partialResultMap` is 0, then just treat it as the first input, so we don't need maintain the status of `isFirstInput`, and `handleFisrtInput` can also be removed which makes the code easier to read.

done